code/modules/clothing/modular_armor/attachments/modules.dm

```diff
@@ -590,3 +590,28 @@
 	SIGNAL_HANDLER
 	beacon_datum = null
 
+/obj/item/armor_module/module/artemis
+	name = "Mark 1 Artemis Helmet Module"
```

Comment:
improper/proper, could also make this Mk2 to shorten the name

Reply:
its for consistency between module names